main.js

```diff
@@ -21,6 +21,8 @@ const path = require('path');
 const URL = require('url');
 const config = require('./app/features/config');
 
+const showDevTools = Boolean(process.env.SHOW_DEV_TOOLS) || app.commandLine.hasSwitch('show-dev-tools');
```

Comment:
Wait, this is not going to work. Doesn't `app.commandLine`refer to the Chromium command line args?

Reply:
@saghul true, sorry for that. I updated the PR